app/grandchallenge/archives/migrations/0017_archive_item_delete_permission_granting.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+
+def add_archive_item_delete_permissions(apps, schema_editor):
+    Archive = apps.get_model("archives", "Archive")  # noqa: N806
+    ArchiveItem = apps.get_model("archives", "ArchiveItem")  # noqa: N806
+    ArchiveItemGroupObjectPermission = apps.get_model(  # noqa: N806
+        "archives", "ArchiveItemGroupObjectPermission"
+    )
+    Permission = apps.get_model("auth", "Permission")  # noqa: N806
+
+    queryset = Archive.objects.order_by("-created").prefetch_related(
+        "editors_group", "items"
+    )
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, verbosity=0)

```

Comment:
This seems strange. Would it be better to check `if ArchiveItem.objects.exists()`? Then we know the data migration needs to do work and is being done in a non-empty db.

Reply:
True, that also works!